app/models/user.rb

```diff
@@ -3,4 +3,5 @@ class User < ApplicationRecord
 
   # associations
   has_many :events
+  has_many :event_attendances
```

Comment:
- [ ] We can similarly use `has_many :through` association to get the attended events for a specific user:

```ruby
has_many :attended_events, through: :event_attendances, source: :attendee
```
This will ease the deal with showing events in user show page.

Reply:
Thanks for this tip, although from what I see, this is normally useful in case my join table does not have extra information - just acting as a pass-through, but in case it was storing any useful info, this can not work. But once again thanks for the tip, after implementing it, I saw how it helps in case one has to deal with these join tables.